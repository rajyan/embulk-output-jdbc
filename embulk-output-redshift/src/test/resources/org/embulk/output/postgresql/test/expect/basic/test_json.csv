id:long,json_item:json,jsonb_item:json
1,null,null
2,88,99
3,{"x":"@"},{"y":{"z":[5]}}
4,,
