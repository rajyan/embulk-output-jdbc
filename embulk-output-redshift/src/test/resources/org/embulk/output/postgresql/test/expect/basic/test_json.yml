table: test_json
mode: insert
