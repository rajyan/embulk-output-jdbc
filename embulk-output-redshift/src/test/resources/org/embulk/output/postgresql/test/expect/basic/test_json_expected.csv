1,null,null
2,88,99
3,{"x":"@"},{"y": {"z": [5]}}
4,\N,\N
